src/server/identity/server/auth.go

```diff
@@ -0,0 +1,112 @@
+package server
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"github.com/dexidp/dex/api/v2"
+	"github.com/dexidp/dex/server"
+	"github.com/dexidp/dex/storage"
+	"github.com/dexidp/dex/storage/sql"
+	logrus "github.com/sirupsen/logrus"
+)
+
+const (
+	dexHttpPort = ":658"
+)
+
+type dexServer struct {
+	api.DexServer
```

Comment:
The existing Dex grpc API supports configuring clients but not connectors. If we upstream connector support we may be able to merge this file with `api_server.go`.